content/docs/command-reference/add.md

```diff
@@ -128,10 +128,9 @@ not.
   among the `targets`, this option is ignored. For each file found, a new `.dvc`
   file is created using the process described in this command's description.
 
-- `--no-commit` - do not save outputs to cache. A `.dvc` file is created, while
-  nothing is added to the cache. (`dvc status` will report that the file is
-  `not in cache`.) Use `dvc commit` when ready to commit outputs with DVC. This
-  is analogous to using `git add` before `git commit`.
+- `--no-commit` - do not cache outputs. A `.dvc` file is created, while nothing
+  is stored in the cache. Use `dvc commit` to finish the operation. This is
```

Comment:
@shcheklein 

> `save something to cache` - way more explicit
> repetitive with is stored in the cache

I shortened it because it was repeated/explained in the next sentence indeed. But I've reverted and removed the 2nd mention now (in run/repro too).

> def not outputs here

Well, it's in the `outs` field of the .dvc file but I guess that's an implementation detail and we don't want to mix this up with stages. But, so should I remove all the mentions of "output" in add? There's another 4 (even uses abbr tags).

> Use dvc commit to finish the operation. -> can be used to save ...

I don't think we should explain `dvc commit` here. The user can click and learn about it in it's ref? That way we simplify maintaining the commit explanation too.

Reply:
> But, so should I remove all the mentions of "output" in add?

no, please. This PR is already unmanageable :(

Reply:
> I don't think we should explain dvc commit here.

I think it's fine to mention. Again not sure why do we change it all to be honest - looks very cosmetic and debatable.

Reply:
> not sure why do we change it all to be honest

A contributor changed this part recently and it became evident that it was a mess, as we are discovering more and more now. I've extracted the commit-related stuff to #1989 so we can try to merge this one first and then come back to this without any rush.

Reply:
BTW, on

> > def not outputs here
>
> so should I remove all the mentions of "output" in add?

Actually, while I agree that .dvc file outputs aren't real "outputs" (from a stage) it woudl be pretty hard to remove this connection in other refs. e.g. `dvc remove` states "removes `.dvc` files... and optionally the actual output files they track (see `--outs` option)" (being reviewed in #1971).

This connection between .dvc files and the "output" term is probably pervasive, even in the field name of course (`outs`) so I think it's better to treat outputs as a more general concept, not only for stages or pipelines.